playbooks/ci-full-rhel/tasks/create_heat_template.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: create heat template for test env
```

Comment:
@paulczar I think this is not needed, either. If so , I need to modify setup script or the script will call create_heat_template.yml .